ocs_ci/ocs/fiojob.py

```diff
@@ -548,6 +554,9 @@ def is_storage_reclaimed():
         # Without checksum, we just need to make sure that data were deleted
         # and wait for this to happen to avoid conflicts with tests executed
         # right after this one.
-        delete_fio_data(fio_job_file, is_storage_reclaimed)
+        if not keep_fio_data:
+            delete_fio_data(fio_job_file, is_storage_reclaimed)
+        else:
+            logging.info("The fio data will be deleted at the teardown part")
```

Comment:
Minor suggestion to make it more clear why and when the data are actually deleted in this case:
```suggestion
            logging.info("The fio data will be deleted during project teardown")
```